onmt/decoders/transformer.py

```diff
@@ -12,21 +12,41 @@
 
 
 class TransformerDecoderLayer(nn.Module):
-    """
+    """Transformer Decoder layer block in Pre-Norm style.
```

Comment:
Maybe cite this paper for detail about pre-norm?
https://arxiv.org/pdf/1906.01787.pdf